tests/assets/README.md

```diff
@@ -15,6 +15,12 @@ Kevin MacLeod (incompetech.com),
 licensed under Creative Commons:
 [CC-BY-3.0](http://creativecommons.org/licenses/by/3.0/).
 
+We converted the file `gs-16b-1c-44100hz.opus`
```

Comment:
Is this a typo? Did you mean the .aac or the .m4a one?
My understanding is that you are forcing mono and 
16KHz.  


```
total 1020
-rw-rw-r-- 1 cgeng cgeng 137654 Dez 19 15:37 gs-16b-1c-16000hz.opus
-rw-r--r-- 1 cgeng root  137099 Aug  2 03:09 gs-16b-1c-44100hz.aac
-rw-r--r-- 1 cgeng root  649912 Aug  2 03:09 gs-16b-1c-44100hz.m4a
-rw-r--r-- 1 cgeng root   25350 Aug  2 03:09 gs-16b-1c-8000hz.amr
-rw-rw-r-- 1 cgeng cgeng    916 Dez 19 15:37 README.md
-rw-rw-r-- 1 cgeng cgeng  81873 Dez 19 15:36 video.mp4
```

Reply:
I see now that there is a deletion

deleted    tests/assets/gs-16b-1c-44100hz.opus (binary)

This is because opus uses 16KHZ without telling the user?